### PR TITLE
Test: exclude from total balance

### PR DESCRIPTION
**Pull request by anastasiyaig**
1. https://ethstatus.testrail.net/index.php?/cases/view/703508 is covered (separate commit)
2. existing test for editing status account is a bit polished and has one more verification now (separate commit)

Fixes https://github.com/status-im/desktop-qa-automation/issues/280
CI run: https://ci.status.im/job/status-desktop/job/e2e/job/manual/847/

Screenshot 2023-11-10 at 10 32 01

### CODE REVIEW

File: gui/screens/settings_wallet.py [hunk 1]
```diff
@@ -101,6 +117,13 @@ def is_remove_account_button_visible(self):
     def get_account_name_value(self):
         return self._wallet_account_title.text
 
+    @allure.step('Get account balance value')
+    def get_account_balance_value(self):
+        time.sleep(3)
```
Comment (anastasiyaig) [inline gui/screens/settings_wallet.py:122]: @StateOf-Vlado @Valentina1133 there is a problem over here, when i open account details, the balance is not updated immediately, the app gets it within approximately 3 seconds. I am open to any suggestions how to improve this time.sleep. What i tried - `waitFor ( float(balance != 0.0))` but it does not help much, the test picks up the value immediately and it is always 0 then.

Comment (Valentina1133) [reply gui/screens/settings_wallet.py:122]: I checked, replaced step in test_settings_include_in_total_balance (48-49) with

assert driver.waitFor(lambda: acc_view.get_account_balance_value() != '0,00', configs.timeouts.UI_LOAD_TIMEOUT_MSEC)

It worked for me without time.sleep, did you add timeout there? Can you try it with timeout?

**Review by Valentina1133 — APPROVED** [review]

File: gui/screens/settings_wallet.py (hunk 1 above)
Comment (anastasiyaig) [reply gui/screens/settings_wallet.py:122]: yessss will do that again, not sure why it was not working before for me 🙄

Comment (anastasiyaig) [reply gui/screens/settings_wallet.py]: yes, indeed, i was doing it incorrectly , i was trying to do with variable which was already stupid. Replacing the condition with function call works, thank you @Valentina1133

**Review by StateOf-Vlado — APPROVED** [review]

**Pull request merged**